opm/input/eclipse/Schedule/MSW/WellSegments.hpp

```diff
@@ -140,14 +139,14 @@ namespace Opm {
         const Segment& topSegment() const;
 
         // components of the pressure drop to be included
-        CompPressureDrop m_comp_pressure_drop;
+        CompPressureDrop m_comp_pressure_drop{CompPressureDrop::HFA};
```

Comment:
@GitPaean : Is `HFA` a reasonable initial value for the pressure drop model of a segment?

Reply:
(Only reason I chose it is that it's the first enum value, ie, to make the {} a little bit more explicit)

Reply:
It should be fine. 

If we want to do it totally safely, we likely need to add an `UNDEFINED`, but I do not see it can cause any problem with the this way when we running a case from a DATA file.  Even someone want to use code to generate a MSW well, it is also safe to use HFA as default, since it should be the default behavoir unless specified otherwise.